src/components/RessourceMenu/RessourceMenu.tsx

```diff
@@ -49,29 +53,35 @@ export const RessourceMenu = () => {
             <td>
               <b>{Math.floor(ressources.food)}</b> ({valueSmoothener(buildingOutputs.food)})
             </td>
-            <td></td>
-            <td>{Math.floor(ressources.villager)}</td>
-          </tr>
-          <tr>
             <td>
-              <b>Remaining</b>
+              <b>{Math.floor(ressources.villager)}</b>
             </td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.gold), Math.floor(ressources.gold))} </td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.wood), Math.floor(ressources.wood))}</td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.stone), Math.floor(ressources.stone))}</td>
-            <td>{remainingRessources(valueSmoothener(buildingOutputs.food), Math.floor(ressources.food))}</td>
-            <td></td>
-            <td></td>
           </tr>
+          <>
+            {selected && (
+              <tr className="ressource-menu-extended-items">
+                <td>
+                  <b>⏳</b>
+                </td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.gold), Math.floor(ressources.gold))} </td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.wood), Math.floor(ressources.wood))}</td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.stone), Math.floor(ressources.stone))}</td>
+                <td>{remainingRessources(valueSmoothener(buildingOutputs.food), Math.floor(ressources.food))}</td>
+                <td></td>
+              </tr>
+            )}
+          </>
         </tbody>
+      </table>
+      {selected && (
         <tfoot>
           <tr>
             <td colSpan={7}>
               <button onClick={() => reset()}>Restart</button>
             </td>
           </tr>
         </tfoot>
-      </table>
+      )}
```

Comment:
the <tfoot> (table foot) needs to be part of the table element above. Also, imo here it also doesn't make sense that it is only visible when a tile is selected